config/kpromo-gh.yaml

```
@@ -0,0 +1,11 @@
+# Configurations for copying GitHub releases to GCS buckets
+# Used by https://sigs.k8s.io/promo-tools/cmd/kpromo
+releaseConfigs:
+- org: containernetworking
+  repo: plugins
+  gcsBucket: k8s-staging-cni-plugins
+  releaseDir: releases
+- org: kubernetes-sigs
+  repo: cri-tools
+  gcsBucket: k8s-staging-cri-tools
+  releaseDir: releases
```

How about renaming this to
```suggestion
  path: releases
```
or `bucketPath` or `subPath`?

---

@saschagrunert -- okay to take this as a follow-up? There are a few other YAML fields I'd like to clean up (preferring kebab-case to camelCase), but they'd need to be fixed in the gh2gcs code in sigs.k8s.io/promo-tools.

---

Sounds good to me :)